src/components/Footer/index.js

```diff
@@ -46,16 +47,26 @@ const getBackgroundColor = ({ pathname, contactUsBg, is404 = false }) => {
     return 'greyBg'
   }
 
-  const path = getPathnameWithSlashes(pathname)
-  /**
-   * These pages all have a final white component so we make sure
-   * to add a grey background to give them some distinction
-   */
-  const colors = {
-    greyBg: ['engineering', 'design', 'training ', 'about-us', 'open-source']
+  const GreyBgSubUrls = {
+    greyBg: [
+      'engineering',
+      'design',
+      'training ',
+      'about-us',
+      'open-source',
+      'case-study'
+    ]
   }
 
-  return Object.keys(colors).find(key => colors[key].includes(path))
+  const path = getPathnameWithSlashes(pathname)
+
+  const isGreySubUrl = Object.keys(GreyBgSubUrls).find(key =>
+    GreyBgSubUrls[key].some(subUrl => path.includes(subUrl))
+  )
+
+  if (isGreySubUrl) {
+    return 'greyBg'
+  }
```

Comment:
```
 const colors = {
    greyBg: [
      'engineering',
      'design',
      'training ',
      'about-us',
      'open-source',
      'case-study'
    ]
  }

return Object.keys(colors).find(key =>
    colors[key].some(subUrl => path.includes(subUrl))
  )
```

This return value should be enough as it will return the key of the colors object in which it finds the path within the subUrl. 

At the moment if colors is something like  `const colors = { white: 'case-study' }` and the path is `case-study/name-of-study` then the conditional `if (isGreySubUrl) { return 'greyBg' }` will be true and make the background colour of the contact us will be `greyBg`

Reply:
oh yeah. I thought we already did that. Gonna update.